cmd/neofs-node/morph.go

```diff
@@ -99,11 +99,19 @@ func listenMorphNotifications(c *cfg) {
 		endpoints[i], endpoints[j] = endpoints[j], endpoints[i]
 	})
 
+	fromSideChainBlock, err := c.persistate.UInt32(persistateSideChainLastBlockKey)
+	if err != nil {
+		fromSideChainBlock = 0
+		c.log.Warn("can't get last processed side chain block number", zap.String("error", err.Error()))
+	}
+
 	for i := range endpoints {
 		subs, err = subscriber.New(c.ctx, &subscriber.Params{
-			Log:         c.log,
-			Endpoint:    endpoints[i],
-			DialTimeout: timeout,
+			Log:            c.log,
+			Endpoint:       endpoints[i],
+			DialTimeout:    timeout,
+			RPCInitTimeout: 10 * time.Second,
```

Comment:
what's the point in using 10 seconds timeout? can neo-go finish syncing in that time?

Reply:
Removed in https://github.com/nspcc-dev/neofs-node/pull/798/commits/b52d3fcbf6d8800b0952b784a1abba50134ac2a7